Source/Scene/Multiple3DTileContent.js

```diff
@@ -0,0 +1,539 @@
+          return true;
+        }
+      }
+
+      return false;
+    },
+    set: function (value) {
+      var contents = this._contents;
+      var length = contents.length;
+      for (var i = 0; i < length; ++i) {
+        contents[i].featurePropertiesDirty = value;
+      }
+    },
+  },
+
+  /**
+   * Part of the {@link Cesium3DTileContent} interface.  <code>Multiple3DTileContent</code>
+   * always returns <code>0</code>.  Instead call <code>featuresLength</code> for a specific inner content.
+   * @memberof Multiple3DTileContent.prototype
+   */
+  featuresLength: {
+    get: function () {
+      return 0;
+    },
+  },
+
+  /**
+   * Part of the {@link Cesium3DTileContent} interface.  <code>Multiple3DTileContent</code>
+   * always returns <code>0</code>.  Instead, call <code>pointsLength</code> for a specific inner content.
+   * @memberof Multiple3DTileContent.prototype
+   */
+  pointsLength: {
+    get: function () {
+      return 0;
+    },
+  },
+
+  /**
+   * Part of the {@link Cesium3DTileContent} interface.  <code>Multiple3DTileContent</code>
+   * always returns <code>0</code>.  Instead call <code>trianglesLength</code> for a specific inner content.
+   * @memberof Multiple3DTileContent.prototype
+   */
+  trianglesLength: {
+    get: function () {
+      return 0;
+    },
+  },
+
+  /**
+   * Part of the {@link Cesium3DTileContent} interface.  <code>Multiple3DTileContent</code>
+   * always returns <code>0</code>.  Instead call <code>geometryByteLength</code> for a specific inner content.
+   * @memberof Multiple3DTileContent.prototype
+   */
+  geometryByteLength: {
+    get: function () {
+      return 0;
+    },
+  },
+
+  /**
+   * Part of the {@link Cesium3DTileContent} interface.   <code>Multiple3DTileContent</code>
+   * always returns <code>0</code>.  Instead call <code>texturesByteLength</code> for a specific inner content.
+   * @memberof Multiple3DTileContent.prototype
+   */
+  texturesByteLength: {
+    get: function () {
+      return 0;
+    },
+  },
+
+  /**
+   * Part of the {@link Cesium3DTileContent} interface.  <code>Multiple3DTileContent</code>
+   * always returns <code>0</code>.  Instead call <code>batchTableByteLength</code> for a specific inner content.
+   * @memberof Multiple3DTileContent.prototype
+   */
+  batchTableByteLength: {
+    get: function () {
+      return 0;
+    },
+  },
+
+  innerContents: {
+    get: function () {
+      return this._contents;
+    },
+  },
+
+  readyPromise: {
+    get: function () {
+      return this._readyPromise.promise;
+    },
+  },
+
+  tileset: {
+    get: function () {
+      return this._tileset;
+    },
+  },
+
+  tile: {
+    get: function () {
+      return this._tile;
+    },
+  },
+
+  /**
+   * Get an array of the inner content URLs, regardless of whether they've
+   * been fetched or not. This is intended for use with
+   * {@link Cesium3DTileset#debugShowUrl}.
+   *
+   * @type {String[]}
+   * @readonly
+   * @private
```

Comment:
For organizational purposes could the getters unique to `Multiple3DTileContent` and not part of the `Cesium3DTileContent` interface be included last?